packages/wonder-blocks-clickable/src/components/__tests__/clickable-behavior.test.js

```diff
@@ -93,18 +93,33 @@ describe("ClickableBehavior", () => {
             </ClickableBehavior>,
         );
         const button = screen.getByRole("button");
+        expect(button).not.toHaveTextContent("hovered");
+        expect(button).not.toHaveTextContent("pressed");
+
+        fireEvent.mouseEnter(button, {buttons: 1});
+        expect(button).toHaveTextContent("hovered");
+    });
+
+    it("changes only pressed state on mouse leave while dragging", () => {
```

Comment:
It looks like the hover state goes away as well when dragging the mouse outside the bounds of the button.